_build/Extension-SetIdentityAndVersion.ps1

```diff
@@ -29,7 +29,7 @@ Param(
   [string]$cmdSetGuid,
 
   [Parameter(Mandatory=$False,Position=10)]
-  [string]$targetVsixCommandMenuName = "Windows Template Studio (local)",
```

Comment:
Not sure this file is still needed. If it is we don't use the "(local)" suffix for anything anymore.